docs/assets/templates/.dockstore.yml

```diff
@@ -9,6 +9,11 @@ service:
   author:
   description:
 
+  # `publish` is a service-wide setting and will affect ALL branches/tags; only set this as needed in a main branch.
```

Comment:
maybe `dockstore-wide`
service is like tool or workflow for us

Reply:
I indeed mean service-wide here, just as I say workflow-wide elsewhere in this PR.

Maybe there's a better way to word it?

Reply:
Oh no, I just missed that this is legit the one for services, carry on